### ISSUE TITLE
Check 1.3 Update

### ISSUE THREAD

**Issue by rheak** [prowler-cloud/prowler#231]
Would it make sense to update check 1.3 to only check IAM users that have their password enabled? There are use cases where accounts previously used IAM account for console acess and now only use IAM accounts for Access keys. This use case still lists a last login value even though the user's console password has been disabled which results in a failed finding in the report that cannot be remediated.

```bash
check13(){
  # "Ensure credentials unused for 90 days or greater are disabled (Scored)"
  COMMAND12_LIST_USERS_WITH_PASSWORD_ENABLED=$(cat $TEMP_REPORT_FILE|awk -F, '{ print $1,$4 }' |grep true | awk '{ print $1 }')
  # Only check Password last used for users with password enabled 
  if [[ $COMMAND12_LIST_USERS_WITH_PASSWORD_ENABLED ]]; then
    for i in $COMMAND12_LIST_USERS_WITH_PASSWORD_ENABLED; do
      DATEUSED=$($AWSCLI iam list-users --query "Users[?UserName=='$i'].PasswordLastUsed" --output text $PROFILE_OPT --region $REGION | cut -d'T' -f1)
      HOWOLDER=$(how_older_from_today $DATEUSED)
      if [ $HOWOLDER -gt "90" ];then
        textFail "User \"$i\" has not logged in during the last 90 days "
      else
        textPass "User \"$i\" found with credentials used in the last 90 days"
      fi
    done
  else
      textPass "No users found with password enabled"
  fi
}
```

**Comment by toniblyx** [comment]
Hi @rheak. Ok, now I get the point of your PR, give me some days to test it as well and merge.

**Comment by rheak** [comment]
Thanks @toniblyx !

**Comment by toniblyx** [comment]
Fixed in PR #232
